src/tailwind.css

```diff
@@ -1805,18 +1805,6 @@ video {
   border-radius: 0.5rem;
 }
 
-.rounded-xl {
```

Comment:
Did you run `build:style` after updating styles.css? Not sure why these generated classes were removed

Reply:
I did. We have to run `build:style` to make CSS changes show up, right?

Reply:
took a look at the netlify, everything looks good!